inc/class-s3-uploads-wp-cli-command.php

```diff
@@ -3,6 +3,60 @@
 class S3_Uploads_WP_CLI_Command extends WP_CLI_Command {
 
 	/**
+	 * Verifies the API keys entered will work for writing and deleting from S3.
+	 *
+	 * @subcommand verify
+	 */
+    public function verify_api_keys() {
+
+		S3_Uploads::get_instance(); // Boot
+
+		$upload_dir = wp_upload_dir();
+
+		// The upload file location on the local filesystem
+		$s3_path = $upload_dir['basedir'] . '/' . mt_rand() . '.jpg';
+
+		// Attempt to copy the file to S3
+		WP_CLI::print_value( 'Attempting to upload file '. $s3_path );
+		
+		// Set Error Handler so we can catch it if it fails 
+		set_error_handler( array( $this, 'copy_fail' ) );
```

Comment:
@zacscott I fixed the bug where `copy` would return `false` on fail - so we this should be ok to update now if you merge `master` into our branch.